src/com/maddyhome/idea/vim/common/TextRange.java

```diff
@@ -20,6 +20,10 @@
 
 import org.jetbrains.annotations.NotNull;
 
+/**
+ * @deprecated Please use com.maddyhome.idea.vim.group.visual.VimSelection
+ */
+@Deprecated
```

Comment:
Let's drop it completely if it's our class, it's deprecated, and it's used only in our code base.

Reply:
I've ended up with leaving both classes because `TextRange` has too many usages and it's removing can cause unwanted behavior.